Fix handwritten version check script so it won't miscompile
and correctly identifies that 2.3.0 is bigger than 1.2.9-beta7
https://bugs.gentoo.org/948927
--- a/check
+++ b/check
@@ -89,6 +89,7 @@
 echo "checking for pcsc-lite version >= $ac_version... " >&5
 cat > conftest.${ac_ext} << EOF
 #include <string.h>
+#include <stdlib.h>
 #include <$ac_hdr>
 int main() {
 if (strcmp(PCSCLITE_VERSION_NUMBER, $ac_version) < 0)
